test/runtests.jl

```diff
@@ -0,0 +1,54 @@
+using Test
+
+function is_test_folder(f, path_separator)
+  a = split(f, path_separator)
+  dir = join(a[1:end-1], path_separator)
+  d = split(dir, path_separator)[end]
```

Comment:
There will be a `splitpath` in future versions of julia https://github.com/JuliaLang/julia/issues/24477

Reply:
Thanks, I wasn't crazy about this function-- I'll update this in my next PR.

Reply:
no problem; also see the `isdir` function.

Reply:
also `basename`